### PR TITLE
Added inheriance parent actions without forbidden actions.

### PR DESCRIPTION
**Pull request by WorstOfAny**

### CODE REVIEW

File: lib/flame/controller.rb [hunk 1]
```diff
@@ -220,7 +222,11 @@ def self.extended(ctrl)
 			end
 
 			def define_parent_actions
-				superclass.actions.each do |public_method|
+				forbidden_actions =
+					if defined?(self::FORBIDDEN_ACTIONS) then self::FORBIDDEN_ACTIONS
+					else []
+					end
```
Comment (AlexWayfer) [inline lib/flame/controller.rb]: ## The first option

```ruby
forbidden_actions =
  defined?(self::FORBIDDEN_ACTIONS) ? self::FORBIDDEN_ACTIONS : []
```

## The second option

```ruby
forbidden_actions =
  begin const_get(:FORBIDDEN_ACTIONS); rescue NameError; []; end
```

## The third option

```ruby
class Controller
  extend Forwardable

  FORBIDDEN_ACTIONS = [].freeze

  # ...
```

File: spec/unit/controller_spec.rb [hunk 2]
```diff
@@ -381,10 +381,14 @@ class ControllerApplication < Flame::Application
...
-		it 'should define actions from parent' do
+		it 'should define actions from parent without forbidden actions' do
 			inherited_controller = Class.new(ControllerController)
+			inherited_controller::FORBIDDEN_ACTIONS = %i[
+				current_reroute hash_reroute index_reroute execute_reroute hooks_reroute
+			].freeze
 			inherited_controller.extend Flame::Controller::ParentActions
-			inherited_controller.actions.should.equal ControllerController.actions
+			inherited_controller.actions
+				.should.equal %i[foo bar baz object_hash respond_for_reroute]
```
Comment (AlexWayfer) [inline spec/unit/controller_spec.rb]: Why not add a second test where there is no such constant?

Comment (WorstOfAny) [inline spec/unit/controller_spec.rb]: Added.

File: lib/flame/controller.rb (hunk 1 above)
Comment (WorstOfAny) [inline lib/flame/controller.rb]: Changed.

**Pull request merged**